pipeline/reference.py

```diff
@@ -0,0 +1,189 @@
+class CoordinateReference(dj.Lookup):
+    definition = """
+    coordinate_ref: varchar(32)
+    """
+    contents = [['lambda'], ['bregma']]
+    
+    
+@schema
+class Device(dj.Lookup):
+    definition = """ # This table contain information about the devices used for recording, or stimulation (photo or electrical). Ideally a device-type object (electrode, laser, etc.) 
+    device_name: varchar(32)
+    ---
+    device_desc = "": varchar(1024)
+    """        
+    
+    
+@schema
+class AnimalSource(dj.Lookup):
+    definition = """
+    animal_source: varchar(32)      # source of the animal, Jax, Charles River etc.
+    """
+    contents = [['Jackson'], ['Homemade']]
+
+
+@schema
+class VirusSource(dj.Lookup):
+    definition = """
+    virus_source: varchar(64)
+    """
+    contents = [['UNC'], ['UPenn'], ['MIT'], ['Stanford'], ['Homemade']]
+
+
+@schema
+class ProbeSource(dj.Lookup):
+    definition = """
+    probe_source: varchar(64)
+    ---
+    number_of_channels: int
+    """
+    contents = [
+        ['Cambridge NeuroTech', 64],
+        ['NeuroNexus', 32]
+    ]
+
+
+@schema
+class Virus(dj.Lookup):
+    definition = """
+    virus: varchar(64) # name of the virus
+    ---
+    -> VirusSource
+    virus_lot_number="":  varchar(128)  # lot numnber of the virus
+    virus_titer=null:       float     # x10^12GC/mL
+    """
+#    contents = [
+#        {'virus_name': 'AAV2-hSyn-hChR2(H134R)-EYFP', 
+#         'virus_source_name': 'UNC'
+#        }
+#    ]
+
+
+@schema
+class Experimenter(dj.Lookup):
+    definition = """
+    experimenter: varchar(64)
+    """
+    contents = [['Nuo Li']]
+
+
+@schema
+class WhiskerConfig(dj.Lookup):
+    definition = """
+    whisker_config: varchar(32)
+    """
+    contents = [['full'], ['C2']]
+       
+    
+#@schema
+#class TrialType(dj.Lookup):
+#    definition = """ # An internal lookup table to encode/decode multiple trial-type labels describing a particular trial (e.g. a trial can be 'No Stim', 'Lick L trial' and 'Incorrect L')
+#    trial_type: varchar(64)
+#    ---
+#    trial_type_code: int
+#    """
+#    contents = [
+#            ['No Stim',0],
+#            ['Stim',1],
+#            ['Lick R trial',2],
+#            ['Lick L trial',3],
+#            ['Stim trials',4],
+#            ['No response',5],
+#            ['Early lick',6],
+#            ['Incorrect L',7],
+#            ['Incorrect R',8],
+#            ['Correct L',9],
+#            ['Correct R',10]
+#            ]  
+    
+
+@schema
+class BehavioralType(dj.Lookup):
+    definition = """
+    behavior_acquisition_type: varchar(64)
+    """    
+    contents = [
+            ['thetaAtBase'],
+            ['ampitude'],
+            ['phase'],
+            ['setpoint'],
+            ['thetafilt'],
+            ['deltaKappa'],
+            ['touch_onset'],
+            ['touch_offset'],
+            ['distance_to_pole'],
+            ['pole_available'],
+            ['beam_break_times']
+            ]
+    
+    
+@schema
```

Comment:
I find these ways of categorizing extracellular and intracellular recordings a bit weird.